demo/src/main/java/com/orange/ods/demo/ui/components/chips/VariantChip.kt

```diff
@@ -161,10 +163,15 @@ private fun getChipText(chipType: ChipType): String {
 @ExperimentalMaterialApi
 @Composable
 private fun ChoiceChip(index: Int, customizationState: ChipCustomizationState) {
+    val selected = customizationState.choiceChipIndexSelected.value == index
     OdsChip(
+        modifier = Modifier.toggleable(
+            value = selected,
+            role = Role.RadioButton,
+            onValueChange = { customizationState.selectChoiceChip(index) }),
         text = "${getChipText(chipType = customizationState.chipType.value)} $index",
-        onClick = { customizationState.selectChoiceChip(index) },
-        selected = customizationState.choiceChipIndexSelected.value == index,
+        onClick = { },
```

Comment:
Selection does not work anymore when Tallback is disabled. I had to set both `onValueChange` and `onClick` to make it work in both cases:

```kotlin
@ExperimentalMaterialApi
@Composable
private fun ChoiceChip(index: Int, customizationState: ChipCustomizationState) {
    val selected = customizationState.choiceChipIndexSelected.value == index
    val onClick: () -> Unit = { customizationState.selectChoiceChip(index) }
    OdsChip(
        modifier = Modifier.toggleable(
            value = selected,
            role = Role.RadioButton,
            onValueChange = { onClick() }),
        text = "${getChipText(chipType = customizationState.chipType.value)} $index",
        onClick = onClick,
        selected = selected,
        outlined = customizationState.outlinedChecked.value,
        enabled = !customizationState.disabledChecked.value,
    )
    Spacer(modifier = Modifier.size(dimensionResource(id = R.dimen.ods_spacing_xs)))
}
```